GUI/Main.Designer.cs

```diff
@@ -481,7 +476,7 @@ private void InitializeComponent()
             this.ModList.ColumnHeadersHeightSizeMode = System.Windows.Forms.DataGridViewColumnHeadersHeightSizeMode.AutoSize;
             this.ModList.Columns.AddRange(new System.Windows.Forms.DataGridViewColumn[] {
             this.Installed,
-            this.UpdateCol,
+            this.Update,
```

Comment:
Please revert, or expand name to UpdateColumn or UpdateSelection or some such. "Update" gives us build warnings.